tests/foreman/api/test_activationkey.py

```diff
@@ -297,6 +297,8 @@ def setUpClass(cls):
         """Create an organization and an activation key belonging to it."""
         cls.org = entities.Organization().create()
         cls.act_key = entities.ActivationKey(organization=cls.org).create()
+        if rm_bug_is_open(4638):
+            cls.act_key.read()  # Wait for elasticsearch to index new act key.
```

Comment:
@sthirugn For some context, see [`ActivationKey.read_raw`](https://github.com/SatelliteQE/nailgun/blob/master/nailgun/entities.py#L221-L238). There's no way to know whether a 404 indicates that an entity doesn't exist or whether it indicates that elasticsearch is catching up. Reading the activation key here stops test execution for a while with the hope that elasticsearch will catch up.